apps/wallet-mobile/src/features/SetupWallet/SetupWalletNavigator.tsx

```diff
@@ -37,6 +37,12 @@ export const SetupWalletNavigator = () => {
       <Stack.Screen
         name="setup-wallet-choose-setup-type"
         component={ChooseSetupTypeScreen}
+        options={{title: strings.addNewWalletTitle}}
+      />
+
+      <Stack.Screen
+        name="setup-wallet-choose-setup-type-init"
+        component={ChooseSetupTypeScreen}
         options={{title: strings.addNewWalletTitle, headerLeft: () => null}}
```

Comment:
dropping the left button can be accomplished by changing the screen options when mounting it, any particular reason to keep 2 routes?

Reply:
I thought that this way the route would be better defined because both paths are accessed from different screens.

My question was how to determine what is the case for hiding or not within the screen with that feature. Also I thought it would make it a little difficult to understand the different routes